cypress/page/menu-content.page.ts

```diff
@@ -3,7 +3,7 @@ class MenuContentPage {
   private menuContentPageURL: string;
 
   constructor() {
-    this.tShirtMenu = '#block_top_menu > ul > li:nth-child(3) > a';
+    this.tShirtMenu = '.menu-content > li:nth-child(3) > a';
```

Comment:
what needs to be changed is the usage of `:nth-child(3)` which is a bad practice. You can use the original one, without the part that I just mentioned and instead of that, try to use an attribute of the `a` tag to indicate we are referring to T-shirts